bridge-adaptivity/bridge_lti/models.py

```diff
@@ -19,7 +19,7 @@ class LtiParameters(models.Model):
     lis_outcome_service_url = models.CharField(max_length=300,default='')
     lis_result_sourcedid = models.CharField(max_length=300,default='')
     oauth_consumer_key = models.CharField(max_length=300,default='')
-    user_id = models.CharField(max_length=300,default='') # lti user id
+    user_id = models.CharField(max_length=300,default='') # bridge_lti user id
```

Comment:
Here and in all over the file separate inline comment with two spaces from the commented line.